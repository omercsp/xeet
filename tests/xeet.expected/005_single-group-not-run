Starting run
Running groups: not_run

Running 004_bad_desc                                         .......[Not Run]
Schema validation error: Additional properties are not allowed ('blat' was unexpected)

Running 008a_bad_command_not_run                             .......[Not Run]
[Errno 2] No such file or directory: 'no'

Running 015b_pre_command_fail                                .......[Not Run] Pre run command returned: 1

Finished iteration (#0/0)
Not ran: 004_bad_desc, 008a_bad_command_not_run, 015b_pre_command_fail

