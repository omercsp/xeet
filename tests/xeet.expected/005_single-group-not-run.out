Starting run
Included groups: not_run
Running tests: 004_bad_desc, 008a_bad_command_not_run, 015b_pre_command_fail

004_bad_desc                                                 ....... [Not Run]
Schema validation error: Additional properties are not allowed ('blat' was unexpected)

008a_bad_command_not_run                                     ....... [Not Run]
[Errno 2] No such file or directory: 'no'

015b_pre_command_fail                                        ....... [Not Run] Pre run command RC=1

Finished iteration (#0/0)
Not ran: 004_bad_desc, 008a_bad_command_not_run, 015b_pre_command_fail

